pyverbs/examples/rc_pingpong.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: (GPL-2.0 OR Linux-OpenIB)
+# Copyright (c) 2019 Mellanox Technologies, Inc. All rights reserved. See COPYING file
+import datetime
+import sys
+
+from pyverbs.qp import QPCap, QPInitAttr, QPAttr, QP
+from pyverbs.pyverbs_error import PyverbsUserError
+from pyverbs.addr import AHAttr, GlobalRoute, GID
+from pyverbs.examples.common import PingPong
+from pyverbs.device import AllocDmAttr, DM
+from pyverbs.wr import SGE, RecvWR, SendWR
+from pyverbs.cq import CQ, CompChannel
+from pyverbs.mr import MR, DMMR
+import pyverbs.enums as e
+
+
+class RCPingPong(PingPong):
+    def __init__(self, logger, args):
+        super(RCPingPong, self).__init__(logger)
+        self.args = None
+        self.parse(args)
+        self.qp = None
+        self.cq = None
+        self.dm = None
+        self.cc = None
+        self.recv_sge = None
+        self.is_server = False
+        self.num_cq_events = 0
+
+    def parse(self, args):
+        self.parser.add_argument('-m', '--mtu', dest='path_mtu', default=1024,
```

Comment:
Please no - I hate this rc_pingpoing model we have already, do not copy it.

This stuff needs to be done as proper tests under the run tests.

Reply:
I agree that we need tests with traffic.
What's wrong with an example that shows how to use pyverbs from scratch?

Reply:
Because you are not giving an example, but reimplement C bad example to be in Python.

A person who is searching for a specific example needs to see clear flow without endless irrelevant options.

Examples are supposed to be small and concrete.

Reply:
I removed the examples-related patches.